src/web/file.rs

```diff
@@ -5,12 +5,14 @@ use crate::{db, error::Result};
 use iron::{status, Handler, IronError, IronResult, Request, Response};
 use postgres::Connection;
 
+const MAX_FILE_SIZE: usize = 5 * 1024 * 1024; // 5MB
+
 pub(crate) struct File(pub(crate) db::file::Blob);
 
 impl File {
     /// Gets file from database
     pub fn from_path(conn: &Connection, path: &str) -> Result<File> {
-        Ok(File(db::file::get_path(conn, path)?))
+        Ok(File(db::file::get_path(conn, path, MAX_FILE_SIZE)?))
```

Comment:
Maybe we should use an environmental variable so we can adjust the limit on the fly instead of having to redeploy

Reply:
Hmm, I don't think an environment variable is strictly needed, and implementing it correctly might be complex (I'd prefer not to read and parse it every single request).